packages/fxa-auth-server/lib/metrics/glean/server_events.ts

```diff
@@ -12,6 +12,12 @@ import mozlog, { Logger } from 'mozlog';
 
 const GLEAN_EVENT_MOZLOG_TYPE = 'glean-server-event';
 type LoggerOptions = { app: string; fmt?: 'heka' };
+type Event = {
+  category: string;
+  name: string;
+  extra?: Record<string, any>;
```

Comment:
The Glean reference says that the values are also strings.  Is there a reason you use `any` here instead?

Reply:
Well, not you, glean_parser.  😀

Reply:
Oh, looks like parser does not yet support specific extra types. Per [docs](https://mozilla.github.io/glean/book/reference/metrics/event.html#extra-metric-parameters) this could be `string`, `boolean`, `quantity`. I'm adding this to my list of issues to look into.